pages/linux/hyprctl.md

```diff
@@ -0,0 +1,32 @@
+# hyprctl
+
+> Utility for controlling parts of Hyprland from a CLI or a script.
```

Comment:
```suggestion
> Control parts of the Hyprland Wayland compositor.
```
We avoid, in general, generic parts of a description. For this, we:
- Try to start the descriptions with a verb in the imperative mood
- Avoid words like "utility" and mentions of where it can be run when it's a CLI, although some descriptions mention when it has a GUI or TUI because that's a special case.

Reply:
Thanks for letting me know. Will keep in mind for future PRs.